lib/plugin/screenshotOnFail.js

```diff
@@ -131,4 +131,16 @@ module.exports = function (config) {
       }
     }, true);
   });
+
+  function _getUUID(test) {
+    if (test.uuid) {
+      return test.uuid;
+    }
+
+    if (test.ctx) {
```

Comment:
do we need check for test: `test.ctx  && test.ctx.test`?

Reply:
I'm not sure if we have the case `test.ctx` is not undefined and `test.ctx.test` is `undefined`?